app/src/main/java/com/ciandt/dragonfly/example/components/chips/ChipsView.kt

```diff
@@ -0,0 +1,140 @@
+package com.ciandt.dragonfly.example.components.chips
+
+import android.content.Context
+import android.content.res.Resources
+import android.graphics.drawable.Drawable
+import android.support.v7.widget.LinearLayoutManager
+import android.support.v7.widget.RecyclerView
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.RelativeLayout
+import com.ciandt.dragonfly.example.R
+import kotlinx.android.synthetic.main.component_chips_view.view.*
+
+
+class ChipsView : RelativeLayout {
+
+    private val DEFAULT_EDGES_WIDTH = dpToPx(16)
+    private val DEFAULT_SPACE_BEFORE = dpToPx(0)
+    private val DEFAULT_SPACE_BETWEEN = dpToPx(0)
+    private val DEFAULT_SPACE_AFTER = dpToPx(0)
+
+    private val chips = ArrayList<Chip>()
+
+    private lateinit var recyclerView: RecyclerView
+    private lateinit var adapter: ChipAdapter
+
+    private var selectCallback: ((Chip) -> Unit)? = null
+    private var deselectCallback: ((Chip) -> Unit)? = null
+
+    constructor(context: Context) : super(context) {
+        initializeViews(context)
+    }
+
+    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {
+        initializeViews(context)
+        initializeAttributes(attrs)
+    }
+
+    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
+        initializeViews(context)
+        initializeAttributes(attrs)
+    }
+
+    fun initializeViews(context: Context) {
+        val inflater = context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
```

Comment:
What about adding an extension function to Context?

`context.getLayoutInflaterService()`